### PR TITLE
Update js packages

### PR DESCRIPTION
**Pull request by khcr**
I updated all the javascript dependencies.
In particular:
- Mapbox from v1 to v2: the breaking change is in the pricing, but I think it is impacting us.
- Stimulus from v1 to v3: the breaking change is with the target definition: `data-target="controller.element"` becomes `data-controller-target="element"`
- Webpacker: latest version of webpacker and webpacker dev server along with latest config files. I changed the way mapbox is excluded from babel transpilation using the official documentation.

@nkcr please test that the javascript and the features are working in your environment.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
   },
-  "version": "0.2.0",
   "devDependencies": {
+    "@babel/plugin-proposal-private-methods": "^7.16.0",
+    "@rails/activestorage": "^6.0.0",
+    "@rails/ujs": "^6.0.0",
+    "@rails/webpacker": "5.4.3",
+    "webpack": "^4.46.0",
+    "webpack-cli": "^4.3.1",
+    "@webpack-cli/serve": "^1.6.0",
```
Comment (khcr) [inline package.json:17]: These are not development dependencies !

Comment (nkcr) [reply package.json:17]: True for `@rails/activestorage` and `@rails/ujs`. But the the others concern the bundling and such, which is not needed in the production app. Or am I missing something ?

Comment (khcr) [reply package.json:17]: Looking at the default package.json in a Rails app, there are not under devDependencies. If we have another insight showing that it is safe to do so, go ahead.

Comment (nkcr) [reply package.json]: ok let's stick to common practice.

**Review by nkcr — APPROVED** [review]

**Pull request merged**